src/404.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>404 Not Found | Caselaw Access Project</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="robots" content="noindex" />
+		<meta name="author" content="Your name" />
+		<meta name="description" content="Explore American Caselaw" />
+		<meta property="og:title" content="Caselaw Access Project" />
+		<meta property="og:description" content="Explore American Caselaw" />
+		<meta property="og:url" content="https://case.law/" />
+		<meta property="og:type" content="article" />
+		<meta property="og:site_name" content="Caselaw Access Projects" />
+
+		<link href="css/global.css" rel="stylesheet" />
+		<script type="module" src="components/cap-nav.js"></script>
+		<script type="module" src="components/cap-page-header.js"></script>
+		<script type="module" src="components/cap-footer.js"></script>
+
+		<link rel="apple-touch-icon" sizes="152x152" href="images/favicon/apple-touch-icon.png">
+		<link rel="icon" type="image/png" sizes="192x192"  href="images/favicon/android-chrome-144x144.png">
+		<link rel="icon" type="image/png" sizes="96x96" href="images/favicon/favicon.ico">
+		<link rel="icon" type="image/png" sizes="32x32" href="images/favicon/favicon-32x32.png">
+		<link rel="icon" type="image/png" sizes="16x16" href="images/favicon/favicon-16x16.png">
+		<link rel="mask-icon" href="images/favicon/safari-pinned-tab.svg" color="#0075FF">
+		<meta name="msapplication-TileColor" content="#2b5797">
+		<meta name="msapplication-TileImage" content="images/favicon/mstile-150x150.png">
+		<meta name="theme-color" content="#ffffff">
+	</head>
+	<body>
+		<cap-nav></cap-nav>
+		<main id="main" class="l-interiorPage">
+			<header class="u-bg-gray-500 u-col-span-full">
+				<cap-page-header heading="Hmm... can't find that.">
+					<p class="u-text-white u-text-serif">
+						Maybe the last borrower didn't return that link.<br>
+						If you got here through our site or an article we host, please let us know at <a class="u-text-purple-200" href="mailto:info@case.law">info@case.law</a> so we can fix it.
```

Comment:
Tiny note here: typically, with running text like this, I'd create two separate `<p>` elements, instead of using a `<br />` element. 

According to the [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) creating separate paragraphs of text using a `<br>` element is problematic for people who navigate with the aid of screenreaders.